data/prepro.py

```diff
@@ -64,7 +64,7 @@ def encode_vocab(data_toks, ques_toks, ans_toks, word2ind):
 
     return data_toks, ques_inds, ans_inds
 
-def create_data_mats(data_toks, ques_inds, ans_inds, params):
+def create_data_mats(data_toks, ques_inds, ans_inds, params, data_split):
```

Comment:
1) Since `test` split does not have `gt_index` and `answer_options`, I have not computed `answer_index` and `options` for `test` split. 
2) But I am not sure of the purpose of `options_list` and `options_len` and do not know whether or not they should be present in `test` split. For now, I have not included them. Let me know whether to keep them in `test` split or not.